packages/app/src/containers/TextStyleInspector.tsx

```diff
@@ -0,0 +1,257 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Divider } from 'noya-designsystem';
+import { Selectors } from 'noya-state';
+import { useCallback, memo, useMemo } from 'react';
+import useShallowArray from '../hooks/useShallowArray';
+import TextOptionsRow, {
+  SimpleTextDecoration,
+} from '../components/inspector/TextOptionsRow';
+import TextAlignmentRow from '../components/inspector/TextLayoutRow';
+import TextStyleRow from '../components/inspector/TextStyleRow';
+import {
+  useSelector,
+  useApplicationState,
+} from '../contexts/ApplicationStateContext';
+
+export default memo(function TextStyleInspector() {
+  const [, dispatch] = useApplicationState();
+
+  //Todo: Handle multiple texts
+  const seletedText = useShallowArray(useSelector(Selectors.getSelectedText));
+
+  const getTextStyleAttributes = useCallback((layers: Sketch.Text[]) => {
+    const layer = layers[0];
+    const textStyle = layer.style?.textStyle;
+    const encodedAttributes = textStyle?.encodedAttributes;
+    const paragraphStyle = encodedAttributes?.paragraphStyle;
+
+    const attributes = layer.attributedString.attributes;
+
+    const color = {
+      _class: 'color',
+      red: 0.5,
+      blue: 0.5,
+      green: 0.5,
+      alpha: 0.5,
+    } as Sketch.Color;
+
+    return {
+      fontColor: encodedAttributes?.MSAttributedStringColorAttribute || color,
+      fontFamily:
+        encodedAttributes?.MSAttributedStringFontAttribute.attributes.name ||
+        'Arial',
+      fontSize:
+        layers.every((layer) => {
+          const textStyle = layer.style?.textStyle;
+          const encoded = textStyle?.encodedAttributes;
+          return (
+            encoded?.MSAttributedStringFontAttribute.attributes.size ===
+            encodedAttributes?.MSAttributedStringFontAttribute.attributes.size
+          );
+        }) &&
+        attributes.every(
+          (a) =>
+            a.attributes.MSAttributedStringFontAttribute.attributes.size ===
+            encodedAttributes?.MSAttributedStringFontAttribute.attributes.size,
+        )
+          ? encodedAttributes?.MSAttributedStringFontAttribute.attributes.size
+          : undefined,
+      lineHeight:
+        layers.every((layer) => {
+          const textStyle = layer.style?.textStyle;
+          const encoded = textStyle?.encodedAttributes;
+          const paragraph = encoded?.paragraphStyle;
+
+          return (
+            paragraph?.maximumLineHeight === paragraphStyle?.maximumLineHeight
+          );
+        }) &&
+        attributes.every(
+          (a) =>
+            a.attributes.paragraphStyle?.maximumLineHeight ===
+            paragraphStyle?.maximumLineHeight,
+        )
+          ? paragraphStyle?.maximumLineHeight ?? 22
+          : undefined,
+      horizontalAlignment:
+        layers.every((layer) => {
```

Comment:
I can't do this because we have a different default value in some cases. 

I did created the function but it returns bolean. You can see it now and check if this is a good approach.